site/_includes/docs/v1/tech/tutorials/_integrate-add-user.liquid

```diff
@@ -0,0 +1,13 @@
+Next, [log into your FusionAuth instance](http://localhost:9011).
```

Comment:
We should have this open in a new window, otherwise you are taken away from the instructions.

Reply:
I'm now using `[log into your FusionAuth instance](http://localhost:9011){:target="_blank"}`